### DISCUSSION TITLE
[Timeseries] Responsive right legend position

### DISCUSSION THREAD

**Discussion by simPod** [grafana/grafana#68755] [Dashboards, Panels & Visualizations]
When viewing timeseries panels with right legend on mobile, the legend should be moved to bottom (the same way the original Graph panel did that)

<img width="550" alt="image" src="https://github.com/grafana/grafana/assets/327717/2163670d-b6ec-4a48-91ee-94ca2d0fa7d8">

Currently, it's kinda impossible to use Grafana on mobile since nothing really is visibile

**Comment by simPod** [comment]
I managed to hack around it using https://github.com/simPod/grafana/commit/4776afb9ee92a2727096cd3853777f342c6f24a9

**Reply by leeoniya** [reply]
looks sane to me, gonna check with some folks to see if there are any concerns. maybe open a PR with this?

**Reply by dprokop** [reply]
Looks good, one thing to change is to use theme breakpoint rather than a fixed value, sth like this:

```
if (document.body.clientWidth < theme.breakpoints.values.lg) {
```

**Reply by simPod** [reply]
https://github.com/grafana/grafana/pull/69045

**Comment by gelicia** [comment]
The PR doing this was merged, so I am closing this discussion.

**Reply by simPod** [reply]
what PR? Discussions are not a very good tool for tracking it it seems.

**Reply by gelicia** [reply]
https://github.com/grafana/grafana/pull/69045
